machine-learning/IDataView.ipynb

```diff
@@ -0,0 +1,375 @@
+          },
+          "execution_count": 1,
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Define schema with classes\n",
+        "\n",
+        "You also have the option of creating new classes or using existing classes to define your schema. Using the same student data above, you can define the schema as follows:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "public class TestScores\n",
+        "{\n",
+        "\tpublic string StudentName {get;set;}\n",
+        "\tpublic string Scores {get;set;}\n",
+        "}"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Loading data\n",
+        "\n",
+        "You can load data from a flat file either using the TextLoader or LoadFromTextFile methods\n",
+        "\n",
+        "#### Loading data from a TextLoader"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "// Initialize MLContext\n",
+        "var mlContext = new MLContext();"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "// Define TextLoader\n",
+        "var textLoader =\n",
+        "    mlContext.Data.CreateTextLoader(\n",
+        "        columns: new TextLoader.Column[]\n",
+        "        {\n",
+        "            new TextLoader.Column(\"StudentName\",DataKind.String, 0),\n",
+        "            new TextLoader.Column(\"Score\", DataKind.Single, 1)\n",
+        "        },\n",
+        "        separatorChar: ',',\n",
+        "        hasHeader: true);"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "// Create IDataView\n",
+        "var textLoaderDataView = textLoader.Load(\"student-scores.csv\");"
```

Comment:
Can you move this file under machine-learning/data?

Reply:
Done